src/pages/Main/Main.jsx

```diff
@@ -1,7 +1,169 @@
-import React from "react";
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  BiCalendar,
+  BiSolidUserCircle,
+  BiBell,
+  BiBarChartAlt,
+} from 'react-icons/bi';
+import './Main.scss';
+
+const accessToken = localStorage.getItem('accessToken');
+const userName = localStorage.getItem('userName');
 
 const Main = () => {
-  return <div>Main</div>;
+  const navigate = useNavigate();
+
+  const handleLogout = () => {
+    localStorage.removeItem('accessToken');
+    localStorage.removeItem('userName');
+    localStorage.removeItem('phoneNumber');
+    localStorage.removeItem('profileImage');
+    navigate('/login');
+  };
+
+  useEffect(() => {
+    if (!accessToken) {
+      alert('로그인 후 이용해주세요.');
+      navigate('/login');
+    }
+  }, []);
+
```

Comment:
비로그인 시 해당 페이지의 내용을 확인할 수도 없게 막으려면, 토큰이 없는 경우에 이렇게 추가로 처리해야 할 것 같습니다.
```suggestion

  useEffect(() => {
    if (!accessToken) {
      alert('로그인 후 이용해주세요.');
      navigate('/login');
    }
  }, []);

  if (!accessToken) return null;

```